runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -2180,6 +2175,19 @@ static void jitHookClassLoaderUnload(J9HookInterface * * hookInterface, UDATA ev
 
 #endif /* defined (J9VM_GC_DYNAMIC_CLASS_UNLOADING)*/
 
+#if defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING) && defined(J9VM_JIT_DYNAMIC_LOOP_TRANSFER)
+static void jitHookClassLoadersUnload(J9HookInterface * * hookInterface, UDATA eventNum, void * eventData, void * userData)
+   {
+   J9VMClassUnloadEvent * unloadedEvent = (J9VMClassUnloadEvent *)eventData;
+   J9VMThread * vmThread = unloadedEvent->currentThread;
+   J9JITConfig * jitConfig = vmThread->javaVM->jitConfig;
+   TR::CompilationInfo * compInfo = TR::CompilationInfo::get(jitConfig);
+      compInfo->cleanDLTRecordOnUnload();
+      if (compInfo->getDLT_HT())
+         compInfo->getDLT_HT()->onClassUnloading();
```

Comment:
I don't understand the indentation here.  Why is `compInfo->....` more deeply nested than the variable declaration above?

Reply:
Thanks for catching that. 
See: 72c265fb0373f36991c94532768c471b29665784